src/kokkos/KokkosCore/HistoContainer.h

```diff
@@ -8,7 +8,8 @@
 
 #include <Kokkos_Core.hpp>
 
-#include "KokkosCore/AtomicPairCounter.h"
+#include "AtomicPairCounter.h"
+#include "kokkos_assert.h"
```

Comment:
The `kokkos/OneToManyAssoc_t.cuda` comes from `kokkos_assert.h`. Or really the logic inside `kokkos_assert.h`, the standard `<cassert>` itself works. I'm exploring some ways how deal with that (at the system level)

Reply:
#77 should provide a proper fix.

Reply:
Yup, that did it, the unit tests work now with this PR.